pyproject.toml

```diff
@@ -32,15 +39,16 @@ dev = [
     'furo'
 ]
 
+
 [tool.setuptools.packages.find]
-where = ["src"]
+where = ["src", "examples"]
 
 [tool.setuptools.dynamic]
 version = {attr = "aind_behavior_curriculum.__version__"}
 
 [tool.black]
 line-length = 79
-target_version = ['py36']
+target_version = ['py310']
```

Comment:
311 maybe? Since 310 doesnt have the pipe operators we mentioned before?

Reply:
Bumped to 3.11+